jenkinsfiles/ios_jenkinsfile.groovy

```diff
@@ -84,4 +92,4 @@ node('ios') {
     stage('Archive') {
         archiveArtifacts "build/Debug-iphoneos/${OUTPUT_FILE_NAME}"
```

Comment:
very good information here!

so, @odra I understand that we need to modify the plugin to accept that parameter. We need to change the Jenkinsfile to pass that parameter from Jenkins build parameter. Let's merge this one as I create issues : 
- https://issues.jboss.org/browse/AGDIGGER-134
- https://issues.jboss.org/browse/AGDIGGER-135

Agree @wei-lee @odra ?